test/run_test.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python3
+
+import subprocess
+
+def run_test():
+	try:
+		subprocess.run(['pytest'], check=True)
+	except subprocess.CalledProcessError as e:
+		print(f"pytest failed with exit code {e.returncode}")
```

Comment:
actions が exit 0 or 1 でテストが通ったかどうか見てる (多分) ので、一旦は pytest が失敗したら exit(1) とかすると良いと思います
(試しにテストコードの assert をわざと失敗するように書いて actions が失敗を通知してくれるか見てみる、などすると分かるかも)
```suggestion
	except subprocess.CalledProcessError as e:
		print(f"pytest failed with exit code {e.returncode}")
		exit(1)
```

Reply:
確かに！だからpytestが失敗してもgithub actionsでは通ったんだ。ありがとうございます！